geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb

```diff
@@ -234,11 +231,11 @@ Start the JDK's Java VisualVM monitoring application in a separate process.
 start jvisualvm [--J=value(,value)*]
 ```
 
-| Name                                      | Description |
-|-------------------------------------------|------------------------------------------------------------------------------------------------|
-| <span class="keyword parmname">\\-\\-J</span> | VM-option passed to the spawned CacheServer VM. For example: `-J-Dfoo.bar=true` for setting foo.bar to 'true'. |
+**Parameters, start jvisualvm**
 
-<span class="tablecap">Table 4. Start JVisalVM Parameters</span>
+| Name       | Description |
+|------------|-------------|
+| <span class="keyword parmname">\\-\\-J</span> | VM-option passed to the spawned CacheServer VM. For example: `-J-Dfoo.bar=true` for setting foo.bar to 'true'. |
```

Comment:
I think the example should begin "--J=-Dfoo..."

Reply:
Fixed.